applications/TheLostDiamond_BjornOmsland.md

```diff
@@ -0,0 +1,208 @@
+
+We are not aware of other similar real-world P2E adventure game projects in the EOSIO or other related ecosystems. 
+
+
+## Team
+
+### Team members
+
+- Team leader: Bjørn Omsland
+- Team members: Ann Magrit Monhof, Siv Anne Balslev, Jørn Balslev
+
+### Contact
+
+- **Contact Name:** Bjørn Omsland
+- **Contact Email:** bomsland@gmail.com
+- **Website:** thelostdiamond.io
+- **Time2Discover Website:** time2discover.no
+
+### Legal Structure
+- **Registered Legal Entity:** Time2Discover AS (Norway) https://w2.brreg.no/enhet/sok/detalj.jsp?orgnr=925474746
+- **Registered Address:** Rosenholmveien 25, 1414 Trollåsen, Norway
+
+### Team Experience
+
+We are four creative outdoor enthusiasts from Norway with backgrounds in the travel industry, logistics, project management and programming.
+
+We have hosted several Lost Diamond events in Norway, utilizing the EOS-Mainnet, for more than 1,000 users to test and improve TheLostDiamond race.
+
+Our programmer (Bjørn Omsland) has twenty years experience as a full stack developer and four year experience as a smart contract developer on the EOS-blockchain. 
+
+
+### Team Code Repos
+
+- https://github.com/bjornomsland
+- https://github.com/bjornomsland/CaptainBlackBillSmartContract
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+N/A
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/bjornomsland/
+- https://www.linkedin.com/in/ann-magrit-monhof-55785610/
+- https://www.linkedin.com/in/jorn-balslev/
+- https://www.linkedin.com/in/sivbalslev/
+
+## Development Status
+
+We have a working MVP that has been used for testing the application in several single-event races in Norway. https://cptblackbill.com/
+
+The smartcontract is located on the EOS-Mainnet account cptblackbill. The code is available at https://github.com/bjornomsland/CaptainBlackBillSmartContract
+
+The frontend is developed in NodeJs and hosted on Google Clouds. 
+
+Different single-event races can have different unique domain names. E.g. our biggest single-event race is “Discover Oslo Adventure Race”. We will host our third Discover Oslo Adventure Race on June 11th. https://opplevoslo.app The feedback from our Discover Oslo participants has been exceptionally good. 98% say they will participate in the upcoming Discover Oslo race.
+
+Development, testing and feedback from all the single-event races is the foundation for our main goal - launching The Lost Diamond 24/7 Global Adventure Race. The Lost Diamond 24/7 Global Adventure Race is ready for Test Marketing, where the goal is to find out how many people will participate in a pure crypto-based adventure race and see if we can obtain a 22% monthly growth rate. 
```

Comment:
Sounds exciting, similar GeoCaching

Curious to know how to prevent users from claiming prizes using multiple fake accounts.

If similar to GeoCaching, once a user finds a prize, couldn't that use create a new fake account and claim that prize again?

Would be interesting to know how abusive behavior is managed within this application

Reply:
Thanks. Yes, similar to GeoCaching, but EOS has the advantage to do this with money.

Multiple fake accounts is not a problem, since that will be additional users paying the $7 dollar monthly subscription fee.

We have worked on the game theory and abusive behavior for a long time. You will find more details about this in the Pitch Deck Appendix (link in the Additional Information section).